v2/robotmk/src/results.rs

```diff
@@ -36,30 +36,31 @@ impl WriteSection for RCCSetupFailures {
 }
 
 #[derive(Serialize)]
-pub struct BuildStates<'a>(pub &'a HashMap<String, EnvironmentBuildStatus>);
+pub struct BuildStates<'a>(pub &'a HashMap<String, EnvironmentBuildStage>);
 
 impl WriteSection for BuildStates<'_> {
     fn name() -> &'static str {
         "robotmk_environment_build_states"
     }
 }
 
-#[derive(Serialize)]
-pub enum EnvironmentBuildStatus {
-    Success(i64),
-    Failure(EnvironmentBuildStatusError),
+#[derive(Serialize, Clone)]
+pub enum BuildOutcome {
```

Comment:
We have to adjust the models in Checkmk accordingly

Reply:
Done